src/App.js

```diff
@@ -16,7 +16,8 @@ function App(props) {
 			<Routes>
 				<Route path='/' element={<Articles />} />
 				<Route path='/articles' element={<Articles />} />
-				<Route path='/articles/:topic' element={<Articles />} />
+				<Route path='/articles/:topic/:sort_by/:order' element={<Articles />} />
```

Comment:
This is leading to some unexpected behaviour when I navigate to the topics pages as the queries are not in order. Topic is find to get from params but I would suggest using useQuery from react-router-dom or useState for the sort and order